requirements-dev.txt

```diff
@@ -1,7 +1,9 @@
 behave==1.2.6
 behaving==2.0.0
 Appium-Python-Client<=0.52
+ckanapi==4.3
+factory-boy==2.12.0
 flake8==3.8.3
-nose==1.3.7
+mock
+pytest-ckan
 splinter>=0.13.0
```

Comment:
need splinter limit

Reply:
Yes, that's now merged from the other pull request.